site/content/contribute/getting-started/_index.md

```diff
@@ -1,24 +1,32 @@
 ---
-title: "Getting Started"
-heading: "Getting Started with Mattermost"
-description: "Learn how you can join the Mattermost community and get started contributing code to Mattermost repositories in GitHub."
+title: "Contribute Code"
+heading: "Contribute Code to Mattermost"
+description: "Learn how to contribute code to the core Mattermost repositories."
 date: "2018-05-19T12:01:23-04:00"
 weight: 1
 ---
 
-Welcome to the Mattermost project!
+This site is for developers who want to contribute code to the core Mattermost project. If you’re looking for other ways to contribute, [head over to our website](https://mattermost.com/contribute/). Before getting started, it’s a good idea to review our guide on [integrating and extending Mattermost](https://developers.mattermost.com/integrate/getting-started/) because you might be able to build the improvements you want without needing to contribute them upstream.
 
-We're very glad you want to join hundreds of community members who have contributed code to Mattermost repositories in GitHub.
 
-Our goal is to make your experience as great as possible. Follow these simple steps to contribute:
+## Technical Overview
 
-{{< bignumber number="1" title="Sign up to our Mattermost site" content="Go to [community.mattermost.com](https://community.mattermost.com/signup_user_complete) and join the [Developers](https://community.mattermost.com/core/channels/developers) and [Contributors](https://community.mattermost.com/core/channels/tickets) channels to ask questions." >}}
+The Mattermost core repositories include:
+* [Server](https://developers.mattermost.com/contribute/server/) - Highly-scalable Mattermost installation written in Go
+* [Web App](https://developers.mattermost.com/contribute/webapp/) - JavaScript client app built on React and Redux
+* [Mobile Apps](https://developers.mattermost.com/contribute/mobile/) - JavaScript client apps for Android and iOS built on React Native
+* [Desktop App](https://developers.mattermost.com/contribute/desktop/) - An Electron wrapper around the web app project that runs on Windows, Linux, and macOS
```

Comment:
This section is titled as "The Mattermost core repositories", but can we have a mention of contributing to plugin repositories on this landing page as well?

Reply:
@mickmister I added a link to /contribute/plugins to this list. That page is light on content right now, but I can improve it in my next PR. I would like to highlight the plugins we maintain a lot more.